docs/reference/requirement-sets/excel-preview-apis.md

```diff
@@ -19,7 +19,6 @@ The first table provides a concise summary of the APIs, while the subsequent tab
 | Document tasks | Turn comments into tasks assigned to users. | [DocumentTask](/javascript/api/excel/excel.documenttask) |
 | Formula changed events | Track changes to formulas, including the source and type of event that caused a change. | [Worksheet.onFormulaChanged](/javascript/api/excel/excel.worksheet#onFormulaChanged)|
 | Linked data types | Adds support for data types connected to Excel from external sources. | [LinkedDataType](/javascript/api/excel/excel.linkeddatatype)|
-| Named sheet views | Gives programmatic control of per-user worksheet views. | [NamedSheetView](/javascript/api/excel/excel.namedsheetview) |
```

Comment:
Can we move this to the Online Only requirement set page? Let's add a similar table to that article (even if this is the only item it contains).